test/integration/requests.test.ts

```diff
@@ -256,4 +256,98 @@ describe("/requests", function () {
         });
     });
   });
+
+  describe("GET /requests", function () {
+    it("should return all requests", function (done) {
+      chai
+        .request(app)
+        .get("/requests?dev=true")
+        .end(function (err, res) {
+          expect(res).to.have.status(200);
+          done();
+        });
+    });
+
+    it("should return all requests by specific user", function (done) {
+      chai
+        .request(app)
+        .get("/requests?dev=true&requestedBy=testUser")
+        .end(function (err, res) {
+          expect(res).to.have.status(200);
+          expect(res.body.data.every((e: any) => e.requestedBy === "testUser"));
```

Comment:
Var name should be more descriptive rather than single character

Reply:
done.